charts/app-config/values-production.yaml

```
@@ -2590,6 +2590,30 @@ govukApplications:
 
   - name: smokey
     chartPath: charts/smokey
+    helmValues:
+      cronJobs:
+        - name: smokey
+          schedule: "*/10 * * * *"
+          args:
+            - cucumber
+            - --profile={{ .Values.govukEnvironment }}
+            - --strict-undefined
+            - --tags=not @notreplatforming and not @not{{ .Values.govukEnvironment }}
+        - name: smokey-failover
+          schedule: "0 9 * * 1-5"
+          args:
+            - env GOVUK_PROXY_PROFILE=failoverCDN
```

https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/

```suggestion
          env:
            - name: GOVUK_PROXY_PROFILE
              value: failoverCDN
          args:
```

---

Oh wait we're still in values.yaml here sorry — you'll need to pass this into the template, but the idea is still the same.